### PR TITLE
Removed some linter warnings

### PR DESCRIPTION
**Pull request by toby3d**
merging variables, struct size, identical expressions and etc.

### CODE REVIEW

File: app_handler.go [hunk 1]
```diff
@@ -37,11 +37,10 @@ func (app *App) handler() func(*fasthttp.RequestCtx) {
 		}
 		app.middlewaresMu.RUnlock()
 		if len(app.domains) > 0 {
-			d := string(ctx.URI().Host())
 			app.domainListLock.RLock()
-			if app.domains[d] != nil {
+			if app.domains[string(ctx.URI().Host())] != nil {
```
Comment (kirillDanshin) [inline app_handler.go]: please revert this change. the `d` variable was used to avoid unnecessary allocations. you can rename it to `domain` if you'd like to, but I think we should keep the var

**Comment by toby3d** [comment]
Travis complains about capitalization in the `apiClient` directory name and breaks testing. I can fix it in the corresponding PR #28

File: app_handler.go [hunk 2]
```diff
@@ -37,11 +37,11 @@ func (app *App) handler() func(*fasthttp.RequestCtx) {
 		}
 		app.middlewaresMu.RUnlock()
 		if len(app.domains) > 0 {
-			d := string(ctx.URI().Host())
+			d := ctx.URI().Host()
```
Comment (kirillDanshin) [inline app_handler.go]: Please revert this change. You forgot to move string()

Comment (toby3d) [reply app_handler.go]: String used in map indexing: https://github.com/toby3d/gramework/blob/da22343551eae224e89a5505277c0d1f25cdb900/app_handler.go#L42-L44

Comment (kirillDanshin) [reply app_handler.go]: Yes, and one call is better then two calls, so we should just declare `d` as a string

File: context.go [hunk 3]
```diff
@@ -164,7 +164,7 @@ func (ctx *Context) CORS(domains ...string) *Context {
 	origins := make([]string, 0)
 	if len(domains) > 0 {
 		origins = domains
-	} else if headerOrigin := ctx.Request.Header.Peek(hOrigin); headerOrigin != nil && len(headerOrigin) > 0 {
+	} else if headerOrigin := ctx.Request.Header.Peek(hOrigin); headerOrigin != nil {
```
Comment (kirillDanshin) [inline context.go]: headerOrigin may be allocated with len(0), so if you trying to minimize CMPQ/JNE's I'd like to leave the len > 0 part instead of != nil

Comment (toby3d) [reply context.go]:
```
context.go:167:62:warning: should omit nil check; len() for nil slices is defined as zero (S1009) (megacheck)
```

File: app_handler.go (hunk 2 above)
Comment (toby3d) [reply app_handler.go]:
```
app_handler.go:40:15:warning: m[string(key)] would be more efficient than k := string(key); m[k] (SA6001) (megacheck)
```

File: context.go (hunk 3 above)
Comment (kirillDanshin) [reply context.go]: Yes, so just replace != nil with len > 0

**Comment by kirillDanshin** [comment]
thanks for your contribution, and welcome to the family 🙂
